spec/lib/email_category_badge_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "email_category_badge"
+
+RSpec.describe EmailCategoryBadge do
+  it "escapes HTML in category names" do
+    c = Fabricate(:category, name: "<b>name</b>")
+
+    html = EmailCategoryBadge.html_for(c)
+
+    expect(html).not_to include("<b>name</b>")
+    expect(html).to include(ERB::Util.html_escape("<b>name</b>"))
```

Comment:
Nitpick, but it might be best to use the escaped text directly here, it would clearer to people reading the code what the expected output would be.